code/datums/dna.dm

```diff
@@ -320,6 +321,7 @@
 //Do not use force_transfer_mutations for stuff like cloners without some precautions, otherwise some conditional mutations could break (timers, drill hat etc)
 	if(newfeatures)
 		dna.features = newfeatures
+		flavor_text = dna.features["flavor_text"] //Update the flavor_text to use new dna text
```

Comment:
OH GOD NO WHY OH FUCK JESUS HOLY SHIT WHY THE FUCK IS FLAVOR TEXT A DNA FEATURE THIS IS CURSED AS FUCK

Reply:
I don't know, does it matter?

Reply:
Because holy shit that's just... why

Reply:
I think the main reason is changelings

Reply:
Ah, yeah that makes sense